ui/DesignSystem/Primitive/Avatar.svelte

```diff
@@ -1,14 +1,48 @@
 <script>
+  import { gql } from "apollo-boost";
+  import { getClient, query } from "svelte-apollo";
+
   import Title from "./Title.svelte";
 
   export let style = null;
 
-  export let imageUrl = `https://avatars.dicebear.com/v2/avataaars/S7oswrhcNJkjzUhNW33S.svg`;
+  // the hierarchy of usage for the following avatars is:
+  // imageUrl > avatarFallback > handle
+  export let imageUrl = null;
+  export let avatarFallback = null; // {emoji: <emoji>, background: {r: <r>, g: <g>, b: <b>}};
+  export let handle = null; // handle for querying the fallback avatar
   export let title = null;
 
   export let variant = "user"; // user | project
   export let size = "regular"; // regular | medium | big
 
+  const GET_AVATAR = gql`
```

Comment:
I thought it would be neater if the Avatar primitive handles all that ie for showing the avatar in the user handle input field, but I could also move the query into the Text Input component once I merged this. That would also give the possibility to delay the query while typing in the input field.

Reply:
I think the primitive should be as dumb as possible and only be concerned with displaying the data passed, then the caller can control when to update these and how.

Reply:
I added a todo

Reply:
It is now extracted